Source/SwiftLintFramework/Rules/PrivateUnitTestRule.swift

```diff
@@ -103,7 +114,19 @@ public struct PrivateUnitTestRule: ASTRule, ConfigurationProviderRule {
         -> [StyleViolation] {
 
             guard kind == .Class else { return [] }
-            let classViolations = validateClass(file, kind: kind, dictionary: dictionary)
+            guard isTestClass(dictionary) == true else { return [] }
```

Comment:
> actually folding the kind == .Class check into isTestClass(_:) would be nice.

This would mean either 1) passing kind to `isTestClass()` or 2) pulling kind out of the dictionary in `isTestClass()` which was already done for us.

Technically `isTestClass()` is already checking `kind == .Class` indirectly by calling `superclass` which does the check (by reading it from the dictionary. I figured I was just being extra-explicit by making the check in `validateFile`, if anything, perhaps this check should just be removed completely? I do personally like the extra check, even if it's unnecessary, but I can go either way.